src/main/resources/application.properties

```diff
@@ -1 +1,2 @@
-
+server.servlet.contextPath=/api/v1
```

Comment:
Hmm, not required but lets say we implement v2 api endpoints one day.

We should make this just /api and update all our endpoints to have v1/ as a prefix in the controllers.